OML4Py LiveLabs - Content updated to remove the word "Zeppelin"

MoitreyeeHazarika opened the pull request:
Content updated to remove "Zeppelin", notebooks updated to remove content on ADB provisioning.

MoitreyeeHazarika commented:
LiveLab updated with the followong changes:

- "Zeppelin notebook" replaced with "OML notebook", both from notebook and lab files.
- Removed step on ADB Provisioning from Lab 1 to "Get Started with LiveLabs"
- Updated notebooks (lab 1) to remove content on ADB prov, OML user creation.

tmcginn reviewed (changes requested):
Moitreyee - 
Have you looked at this on your fork or tested it?
It looks like you are adding get-started-with-livelabs.md to the Free Trial version of the workshop.
Tom

MoitreyeeHazarika commented:
Hi Tom, 
From the LiveLab version, I have removed the content on ADB Provisioning, and added it under a diff lab "Get Started with LiveLab" exclusively for the Freetier version. This is as suggested.  
I have tested my forked copy of both the labs.

Sorry for not mentioning this in my comment earlier. 

Thank you
Moitreyee
